sanity-check/main.py

```diff
@@ -192,13 +193,15 @@ def check_analysis_exists(metadata,submission_song_url):
                 analysis["experiment"]["experimental_strategy"]==metadata.get('experimental_strategy') and \
                 analysis['analysisType']['name']=="sequencing_experiment"\
                 :
-                    sys.exit(
-                        "Sample '%s'/'%s' has an existing published analysis '%s' for experiment_strategy '%s.'"
+                    analysis_exists=True
+                    error_message=\
+                        "Sample '%s'/'%s' has an existing published analysis '%s' for experiment_strategy '%s.'" \
                         % \
                         (metadata.get('submitter_sample_id'),metadata.get('sample_id'),analysis['analysisId'],metadata.get('experimental_strategy'))
-                    )
-                else:
-                    return True
+            if analysis_exists:
+                sys.exit(error_message)
+            else:
+                return True
     else:
         return False
```

Comment:
What's the scenario to `return False`?
One relating question, if there are existence of analysis, do you expect the function `check_analysis_exists` to return True or False?

Reply:
`check_analysis_exists` should evaluate to `TRUE` if no prior `analysis` exists. That being said, the `TRUE` isn't used downstream

No scenario should result in `return False`. I included it just in case. Function should result in a system error or `return True`